### PR TITLE
Update copyright holder of test files to match original committer

### PR DESCRIPTION
**Pull request by Orace**
When missing, add header in test file.

This PR addresses #671

### CODE REVIEW

**Comment by Orace** [comment]
Some name are missing and alias are used instead (sholland and AmbientLion).

File: MoreLinq.Test/CartesianTest.cs [hunk 1]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/CartesianTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/Combinatorics.cs [hunk 2]
```diff
@@ -1,3 +1,19 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/Combinatorics.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/AtLeastTest.cs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #region License and Terms
 // MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
+// Copyright (c) 2015 sholland. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/AtLeastTest.cs]:
```suggestion
// Copyright (c) 2015 "sholland". All rights reserved.
```

File: MoreLinq.Test/ExcludeTest.cs [hunk 4]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/ExcludeTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/InterleaveTest.cs [hunk 5]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/InterleaveTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/LagTest.cs [hunk 6]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/LagTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/LeadTest.cs [hunk 7]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/LeadTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/OrderByTest.cs [hunk 8]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/OrderByTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/PartialSortByTest.cs [hunk 9]
```diff
@@ -1,6 +1,6 @@
 #region License and Terms
 // MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
+// Copyright (c) 2016 Felipe Sateler. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/PartialSortByTest.cs]:
```suggestion
// Copyright (c) 2016 Atif Aziz. All rights reserved.
```

See 13d4e52f60737a0c7658f7d57e89b724c820f0b8.

File: MoreLinq.Test/PartialSortTest.cs [hunk 10]
```diff
@@ -1,6 +1,6 @@
 #region License and Terms
 // MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
+// Copyright (c) 2016 Felipe Sateler. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/PartialSortTest.cs]:
```suggestion
// Copyright (c) 2016 Atif Aziz. All rights reserved.
```

See 13d4e52f60737a0c7658f7d57e89b724c820f0b8.

File: MoreLinq.Test/PartitionTest.cs [hunk 11]
```diff
@@ -1,6 +1,6 @@
 #region License and Terms
 // MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/PartitionTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/PermutationsTest.cs [hunk 12]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/PermutationsTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/RandomSubsetTest.cs [hunk 13]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/RandomSubsetTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/RandomTest.cs [hunk 14]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/RandomTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/RankTest.cs [hunk 15]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/RankTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/RunLengthEncodeTest.cs [hunk 16]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/RunLengthEncodeTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/SegmentTest.cs [hunk 17]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/SegmentTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/SliceTest.cs [hunk 18]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/SliceTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/SortedMergeTest.cs [hunk 19]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/SortedMergeTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

File: MoreLinq.Test/SubsetTest.cs [hunk 20]
```diff
@@ -1,3 +1,20 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2010 AmbientLion. All rights reserved.
```
Comment (atifaziz) [inline MoreLinq.Test/SubsetTest.cs]:
```suggestion
// Copyright (c) 2010 Leopold Bushkin. All rights reserved.
```

**Pull request merged**